### PR TITLE
sphinxcontrib-paverutils: initial add

### PR DESCRIPTION
**Pull request by johannesring**
Sphinxcontrib-paverutils requires paver, so PR #1300 has to go in first.

### CODE REVIEW

**Comment by jakirkham** [comment]
Toggling to restart CIs.

**Comment by johannesring** [comment]
This should be ready to go in now.

File: recipes/sphinxcontrib-paverutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - paver >=1.02.0
+
+test:
+  imports:
+    - sphinxcontrib
+    - sphinxcontrib.paverutils
+
+about:
+  home: https://bitbucket.org/dhellmann/sphinxcontrib-paverutils/
+  license: BSD
```
Comment (jakirkham) [inline recipes/sphinxcontrib-paverutils/meta.yaml]: What version of BSD is this `2-Clause` or `3-Clause`?

Comment (jakirkham) [inline recipes/sphinxcontrib-paverutils/meta.yaml]: Also can optionally add `license_family: BSD`.

File: recipes/sphinxcontrib-paverutils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+{% set version = "1.14.0" %}
+{% set sha256 = "2124c851f96356012ed5c92ce320ca3a842cce8a94ddd2319bcb0835b48b9f80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/sphinxcontrib-paverutils/meta.yaml]: Please change to

`url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`

**Comment by johannesring** [comment]
Thanks @jakirkham, I have addressed your comments now.

**Comment by jakirkham** [comment]
Awesome! Thanks @johannesring .

**Pull request merged**